README.md

```diff
@@ -0,0 +1,378 @@
+![header](https://www.signavio.com/wp-content/uploads/2019/09/product-pages-illustrations-suite-1-1.png)
+
+|                                                   |
+|---------------------------------------------------|
+| DMN & Drools (DRL) Execution Example via Signavio |
+
+[![REUSE status](https://api.reuse.software/badge/github.com/signavio/dmn-and-drools-execution-examples)](https://api.reuse.software/info/github.com/signavio/dmn-and-drools-execution-examples)
+[Signavio Website](https://signavio.com) •
+[Docs](https://docs.signavio.com/)
+
+
+[![build_pipleine_with_maven](https://github.com/signavio/dmn-and-drools-execution-examples/workflows/Java%20CI%20with%20Maven/badge.svg)](https://github.com/signavio/dmn-and-drools-execution-examples/actions?query=workflow%3A%22Java+CI+with+Maven%22)
+
+**Disclaimer:**
+
+> This project shows how the artifacts exported from Signavio Process
+> Manager can be used to execute the decision logic. The execution
+> engine we use in our examples is the one provided by RedHat
+> (<https://www.drools.org/>). For any engine specific questions,
+> consult the documentation provided there.
+
+<div class="contents" backlinks="none">
+
+**Contents**
+
+</div>
```

Comment:
I think this doesn't do anything - not sure if it was supposed to be a table of contents before, in the other file? Maybe you can generate one for the markdown as well, e.g. in IntelliJ

Reply:
There was nothing here, just the headline. I removed the div :)